spec/helpers/articles_helper_spec.rb

```diff
@@ -42,5 +42,25 @@
       expect(helper.article_sanity_check).to be_falsey
     end
   end
+  
```

Comment:
I have to turn that question back around. What _should_ happen in that case? If there aren't ten recent articles to pull from, nothing should happen and no error should be thrown to my knowledge as the ActiveRecord query will return [] as opposed to nil. 

We could print a message if no articles are found ("No cases added yet"). 

Am I on track?